tailcfg/c2ntypes.go

```diff
@@ -102,3 +102,17 @@ type C2NTLSCertInfo struct {
 	// TODO(bradfitz): add fields for whether an ACME fetch is currently in
 	// process and when it started, etc.
 }
+
+// C2NVIPServicesResponse is the response (from node to control) from the
+// /vip-services handler.
+//
+// It returns the list of VIPServices that the node is currently serving with
+// their port info and wether they are active or not. It also returns a hash of
+// the response to allow the control server to detect changes.
+type C2NVIPServicesResponse struct {
+	// VIPServices is the list of VIP services that the node is currently serving.
+	VIPServices []*VIPService
+
+	// Hash is the has of VIPServices to allow the control server to detect changes.
```

Comment:
Typo (`has`), also call out that this is the same value as seen in `Hostinfo.ServicesHash`.

Potentially also name it `ServicesHash` for consistency, although that also feels kind of repetitive, I'm not sure the best choice there.